lib/utils/schema.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const _ = require('lodash')
+const Joi = require('joi')
+const Hoek = require('hoek')
+
+module.exports = function() {
+    const toJoiPostValidatation = function(schema) {
```

Comment:
This is done. As much as it can be done without the details of ```relationships``` and ```links```. Bit confused about ```type```. Seems a duplication of effort since ```URL``` already contains the type